### ISSUE TITLE
Sign Windows Binaries

### ISSUE THREAD

**Issue by Varunram** [Musicoin/desktop#157]
See https://github.com/Musicoin/desktop/issues/126#issuecomment-364266436

**Comment by cryptofuture** [comment]
I hardly doubt is worth the effort, since we have sha256sums anyway.

Though is possible under Linux: https://stackoverflow.com/a/29073957
I could try if someone would provide me with the right certificate for exe file.

**Comment by Varunram** [comment]
Yeah, I mean, not *really* that much of a "want-to" at the moment, more like a "would be nice to have" kind of thing.

**Comment by XhmikosR** [comment]
It's doable; it all depends how much money (since the certs aren't free, but you could get a cheap one) and time one wants to spend on this.

**Comment by Varunram** [comment]
@XhmikosR yeah, apparently there are some firewall issues on Windows which might be resolved with signing, have to take a look at this once the issue with the forum is resolved. (btw, sorry for what happened at the other thread!)

**Comment by etaletai13** [comment]
@XhmikosR -- Whaaaaat did you just say!? A PGP key is what you sign releases with ... Yes. They ARE free. You just make one, set it up as a signing key in GitHub, and ..... 


Ohhhhhkay. I'm just gonna leave this one alone.

**Comment by XhmikosR** [comment]
If you believe you can change how Windows works, good luck.

**Comment by etaletai13** [comment]
Worse - it took me almost a whole day to realize you meant signing in the Windows-hates-open-source kind of way. I'm slipping, I swear ...

**Comment by crypto4techs** [comment]
Thanks for linking this to the issue I posted about VirusTotal false positive and code signing. For the security conscious like me, 0 hits in Virus total and code signing are musts. I can't risk the integrity of my wallet computer to a virus or malicious code. I was initially interested in Musicoin but have stayed away until the wallet scans clean and the code is signed.

**Comment by etaletai13** [comment]
@crypto4techs Do you have the abilities to accomplish this one yourself? If so, please do!! If not, let me know and I'll take it.

**Comment by etaletai13** [comment]
No response, and I'll take it as no takers. I'll claim this and  #243 together. The core reason for signing is that people do not appreciate virus, security, os warning, etc. They aren't directly concerned with the same thing a sha does. Always remember that end users are intentionally not developers, and you do not buy a blank canvas for your wall.

**Comment by Varunram** [comment]
@etaletai13 cool, but I don't think the primary reason for this is people not appreciating virus signs, but that people don't like to work with Windows-ish stuff in general. All the best!

**Comment by etaletai13** [comment]
@Varunram Certainly true! But, just as a road ends up with a sign to say "don't run over the person walking by"; people need what they need! Haha, I definitely get the situation; that's why I'm taking the muddle job.

**Comment by pnomolos** [comment]
I'm willing to help with this if there's a certificate I can sign with.  I've got a couple Windows boxes available to verify on.

**Comment by etaletai13** [comment]
@pnomolos Because we're moving to Electron, we can include package signing with an argument. Currently the only holdup for Windows is a cert, and for Mac is a developer account for the organization; which is simple too. Linux is straight-foreward.

Basically this problem isn't a true problem, but a thing that will be completed when we have the first Electron release, and then easily done for the legacy version (if we are keeping that around)

**Comment by pnomolos** [comment]
@etaletai13 OK!  Well if you need any help from someone with a Windows box (or three) to run/build some stuff, just let me know :)

**Comment by etaletai13** [comment]
@pnomolos Certainly! Also that reminds me that we'll need something in 'yarn commit' that will clean up files edited in Windows. It adds something to utf-8 files that will break on every other system.
